src/app/work-item/work-item-list-entry/work-item-list-entry.component.ts

```diff
@@ -51,9 +52,14 @@ export class WorkItemListEntryComponent implements OnInit {
   loggedIn: Boolean = false;
 
   constructor(private auth: AuthenticationService,
+              private broadcaster: Broadcaster,
               private router: Router,
               private workItemService: WorkItemService,
               private logger: Logger) {
+                 this.broadcaster.on<string>('logout')
```

Comment:
Here is a reference to it in the docs. https://angular.io/docs/ts/latest/guide/lifecycle-hooks.html